dnsproxy.cc.patch

```diff
@@ -0,0 +1,11 @@
+--- pdns/dnsproxy.cc.orig       Thu Oct 30 06:18:22 2014
++++ pdns/dnsproxy.cc    Mon Dec  8 14:55:41 2014
+@@ -169,7 +169,7 @@
+       memcpy(&d,buffer,sizeof(d));
+       {
+         Lock l(&d_lock);
+-#if BYTE_ORDER == BIG_ENDIAN
++#ifdef _LITTLE_ENDIAN
```

Comment:
_LITTLE_ENDIAN comes from /usr/include/sys/isa_defs.h.

If the clause is completely removed, then the i86pc version does not work correctly, i86pc platform being little endian.